packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -84,39 +90,45 @@ function validateProductUpdate(
   return oldOrder !== newOrder;
 }
 
-class StripeHelper {
-  private log: any;
+export class StripeHelper {
+  // Note that this isn't quite accurate, as the auth-server logger has some extras
+  // attached to it in Hapi.
+  private log: Logger;
   private cacheTtlSeconds: number;
   private webhookSecret: string;
-  private plansCacheIsEnabled: boolean;
   private stripe: Stripe;
-  private redis: any;
+  private redis: ioredis.Redis | undefined;
 
   /**
    * Create a Stripe Helper with built-in caching.
    */
-  constructor(log: any, config: any, statsd?: StatsD) {
+  constructor(
+    log: Logger,
+    config: ReturnType<conf['getProperties']>,
+    statsd?: StatsD
+  ) {
     this.log = log;
-    this.cacheTtlSeconds =
-      config.subhub.plansCacheTtlSeconds ||
-      config.subscriptions.cacheTtlSeconds;
+    this.cacheTtlSeconds = config.subhub.plansCacheTtlSeconds;
```

Comment:
Type-checker flagged this upon having config type come through, seems `config.subscriptions.cacheTtlSeconds` does not exist, not sure if/when it last existed.